hail/src/main/scala/is/hail/methods/FilterPartitions.scala

```diff
@@ -5,14 +5,14 @@ import is.hail.expr.ir.functions.{MatrixToMatrixFunction, TableToTableFunction}
 import is.hail.expr.types.{MatrixType, TableType}
 import is.hail.rvd.RVDType
 
-class TableFilterPartitions(parts: Array[Int], keep: Boolean) extends TableToTableFunction {
+case class TableFilterPartitions(parts: Seq[Int], keep: Boolean) extends TableToTableFunction {
```

Comment:
`Array[Int]` uses reference equality, so

```
val tfp1 = TableFilterPartitions(Array(1,2), true)
val tfp2 = TableFilterPartitions(Array(1,2), true)
tfp1 == tfp2
```

would compare false. While I couldn't see anywhere this was directly introducing bugs, it could certainly have done so in the future.

This is a `Seq[Int]` instead of a `IndexedSeq[Int]` because the latter doesn't appear to have an automatic `json4s` serializer/deserializer rule.